### PR TITLE
Added TAU constant.

### PR DESCRIPTION
**Pull request by gmpreussner**
Tau (= 2 * Pi) is much more commonly used than Pi itself and should therefore be included in the standard library. Fore more details on the rationale behind Tau, see: http://tauday.com/

### CODE REVIEW

File: lib/pure/math.nim [hunk 1]
```diff
@@ -44,6 +44,7 @@ when not defined(js) and not defined(nimscript):
 
 const
   PI* = 3.1415926535897932384626433 ## the circle constant PI (Ludolph's number)
+  TAU* = 6.2831853071795864769252868 ## the circle constant TAU (= 2 * PI)
```
Comment (dom96) [inline lib/pure/math.nim]: Setting this to ``2*PI`` should work and would be much nicer.

**Comment by gmpreussner** [comment]
Ok, done. I also fixed the unit test.

**Comment by gmpreussner** [comment]
One potential issue with 2 * PI is that the last digit is not rounded correctly

**Comment by dom96** [comment]
Hopefully that won't be too much of a problem.

**Comment by aaronfranke** [comment]
What is the earliest version of Nim this is included in?

**Comment by SolitudeSF** [comment]
0.14.0

**Pull request merged**